fedimint-server/src/config/mod.rs

```diff
@@ -519,7 +513,9 @@ impl ServerConfig {
             "Distributed key generation has completed successfully!"
         );
 
-        Ok(Ok(server))
+        drop(connections);
```

Comment:
Seems unnecessary. It will be dropped at the end of the scope anyway.

Reply:
Okay, wonder why this was in the original code...